EcpBtBot/MainClass.py

```diff
@@ -1,39 +1,36 @@
 
+import logging
 import os
+from .Conversational_Integration import *
 from .DialogFlow.API import *
 from .DialogFlow.DialogFlowResponse import *
-from .Conversational_Integration import *
+from .DialogFlow.Config import *
+
 
-print("Dire Bonjour pour commencer \n")
 
 dico_users = {}
-#dictionnaire (var globale): pour chaque clé (user), on a un dictionnaire de connexions API en fonction des thèmes
-
-class API_reponse:
-
-    def __init__(self, sujet):
-        if sujet == "sepresenter":
-            self.ai = API_SePresenter()
-        elif sujet == "hotel":
-            self.ai = API_Hotel()
-        elif sujet == "restaurant":
-            self.ai = API_Restaurant()
-        elif sujet == "boulangerie":
-            self.ai = API_Boulangerie()
-
-    def reponseBot(self, message):
-        #user_id = user
-        user_message = message
-        rep = DialogFlowResponse(self.ai._get_json_response(user_message))
-        rep_and_qr=Reponse_And_QuickReplies(rep)
+"""
+dictionnaire (var globale): pour chaque clé (user), on a un dictionnaire de connexions API en fonction des thèmes
+"""
+
+class API_Response(object):
+
+    def __init__(self, subject):
+
+        
+        if subject in tokens:
+            self.ai = API_Connection(subject)
+        else: 
+            logging.critical("Subject non defined")
 
+    def ResponseBot(self, message):
```

Comment:
method / function should not start with capital letters: https://www.python.org/dev/peps/pep-0008/

Reply:
ok